data/coalition/coalition missions.txt

```diff
@@ -1392,3 +1392,271 @@ mission "Arachi Cartoonist 2"
+			choice
+				`	"Alright, if it'll help you relax I'll go with you."`
+					goto shrine
+				`	"Sorry, but I'd rather rest in the ship."`
+			`	She takes a step back, looking down to the bangle again. "Alright. Yes, your responsibility, this is not. Meet you in the spaceport later, I shall."`
+			`	You open the hatch for her, and she heads to one of the Saryds nearby, to talk to them. A few seconds into the conversation and they tense up, crossing their arms and swiping them down, as if refusing something, then run off. She tries again with some of the other Saryds, but continues to be rejected. She looks back at your ship dejectedly, then starts eyeing the other ships docked nearby before heading towards a Kimek Thorn that has just landed. As she talks to the captain, the Kimek seems confused, but after a moment, briefly reenters their ship, returning in a puffy coat to follow Vellorae to the railway hub.`
+			`	Long after the sun has set, the Kimek returns, rushing to their ship while trying to shake off the cold. You ask them about Vellorae, and they tell you she headed for the spaceport to "clear her head."`
+				accept
+			label shrine
+			`	She takes a long, deep breath. "Thank you, Captain. Far outside the village, the site is. Come, show you the way there, I will." You leave your ship and follow her to the railway hub and board one of trains that appears to be one of the oldest and more simple at the hub. It lacks seats for non-Saryds, so you are forced to hold tight to what handrails you can reach.`
+			`	You remain on the train until the very last stop, when the sun is already starting to set. as you depart the train, you see a rural village with just a few dozen buildings and a simple dirt road, where a couple workers battle the snow and ice to keep it clear and easy to traverse. You follow Vellorae down the road, and once you reach a rocky pillar with a hole carved through it, Vellorae turns to the forest heading out on a thin, straight trail. Looking behind you, you see the setting sun fitting inside the pillar's opening.`
+			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks, and a faint light coming from within it.`
```

Comment:
```suggestion
			`	As you follow Vellorae down the trail, you see the trees gradually show more and more signs of a forest fire, with darker, damaged barks, increasingly ashy soil, and many fallen, decomposing portions of the forest. The trail ends abruptly at a large scar scorched through the forest. The piles of ash still present below your feet almost resemble the rest of the planet's otherwise snowy ground, but somehow are kept from mixing with the snow. Whether deliberately cleared by more workers or some other force, you cannot tell. Near the center of the scar you see a small mound with an opening in the side, almost like a "shack" made out of the charred tree trunks and rocks. A faint light comes from within it.`
```